packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -44,7 +48,83 @@ export function buildRootObject() {
 
     await getPromiseForIssuerRecords(issuers);
 
-    const allSeatStagings = new Set();
+    const allSeatStagings = new WeakSet();
+
+    /** @type MakeZCFMint */
+    const makeZCFMint = async (keyword, mathHelperName = 'nat') => {
+      const zoeMintP = E(zoeInstanceAdmin).makeZoeMint(keyword, mathHelperName);
+      const mintyIssuerRecord = await E(zoeMintP).getIssuerRecord();
+      // AWAIT
+      const { brand: mintyBrand } = mintyIssuerRecord;
+      const mintyAmountMath = makeAmountMath(mintyBrand, mathHelperName);
+      // TODO some zcf-side registration of the new issuer
```

Comment:
Whoa, this is a pretty big TODO, I think we should not have merged without this. It will not function correctly if contract code tries to reference this issuer.

Reply:
We just need to update the issuerTable and the instanceRecord.

Reply:
Fixed in next PR. PTAL when it's up.

Reply:
#1392 .
Resolving here.